openedx/adg/admin_dashboard/webinars/api/views.py

```diff
@@ -106,5 +108,8 @@ def post(self, request, pk):
             throw HTTP error code
         """
         webinar_to_publish = get_object_or_404(Webinar, pk=pk)
+        if webinar_to_publish.start_time < now():
+            return Response({'error': WEBINAR_START_TIME_ERROR}, status=HTTP_400_BAD_REQUEST)
```

Comment:
Let's call the translation method over this constant. It will not translate in this case even the translation will exist.